README.adoc

```diff
@@ -164,61 +164,61 @@ annotation on the getter method, which has the same effect.
 
 
-The [hotspot=17-63 file=1]`Spacecraft` class has the following constraints applied:
+The [hotspot=Spacecraft file=1]`Spacecraft` class has the following constraints applied:
 
 * Every destination that is specified needs a name and a positive distance. In Bean 
-Validation 2.0, you can specify constraints on type parameters. The [hotspot=24 file=1]`@NotBlank` 
-and [hotspot=24 file=1]`@Positive` annotations constrain the [hotspot=24 file=1]`destinations` 
+Validation 2.0, you can specify constraints on type parameters. The [hotspot=Map file=1]`@NotBlank` 
+and [hotspot=Map file=1]`@Positive` annotations constrain the [hotspot=Map file=1]`destinations` 
 map so that the destination name is not blank, and the distance is positive. The 
-[hotspot=24 file=1]`@Positive` constraint ensures that numeric value fields are greater 
+[hotspot=Map file=1]`@Positive` constraint ensures that numeric value fields are greater 
 than 0.
 
 * A correctly formatted serial number is required. In addition to specifying the 
 built-in constraints, you can create custom constraints to allow user-defined validation
-rules. The [hotspot=26 file=1]`@SerialNumber` annotation that constrains the 
-[hotspot=27 file=1]`serialNumber` field is a custom constraint, which will be created later.
+rules. The [hotspot=serial-num file=1]`@SerialNumber` annotation that constrains the 
+[hotspot=serialNumber file=1]`serialNumber` field is a custom constraint, which will be created later.
 
-Because you already specified constraints on the [hotspot=11-50 file=0]`Astronaut` bean, 
-the constraints do not need to be respecified in the [hotspot=17-63 file=1]`Spacecraft` 
-bean. Instead, because of the [hotspot=21 file=1]`@Valid` annotation on the field, all 
-the nested constraints on the [hotspot=11-50 file=0]`Astronaut` bean are validated.
+Because you already specified constraints on the [hotspot=Astronaut file=0]`Astronaut` bean, 
+the constraints do not need to be respecified in the [hotspot=Spacecraft file=1]`Spacecraft` 
+bean. Instead, because of the [hotspot=Valid file=1]`@Valid` annotation on the field, all 
+the nested constraints on the [hotspot=Astronaut file=0]`Astronaut` bean are validated.
 
 You can also use bean validation with CDI to provide method-level validation. The
-[hotspot=58-62 file=1]`launchSpacecraft` method on the [hotspot=17-63 file=1]`Spacecraft` 
-bean accepts a [hotspot=58 file=1]`launchCode` parameter, and if the [hotspot=58 file=1]`launchCode` 
-parameter is [hotspot=59 file=1]`OpenLiberty`, the method returns [hotspot=60 file=1]`true` 
-that the spacecraft is launched. Otherwise, the method returns [hotspot=61 file=1]`false`. 
-The [hotspot=58-62 file=1]`launchSpacecraft` method uses both parameter and return value 
-validation. The [hotspot=58 file=1]`@NotNull` constraint eliminates the need to manually 
+[hotspot=launch-Spacecraft file=1]`launchSpacecraft` method on the [hotspot=Spacecraft file=1]`Spacecraft` 
```

Comment:
<img width="1518" alt="image" src="https://user-images.githubusercontent.com/8294670/68786822-6f9b9f00-060e-11ea-9e15-5d0a0feb4d95.png">

Reply:
launchSpaceCraft hotspot not working correctly.

Reply:
Updated.